modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/dependencies/ddm/geolocation.ftl

```diff
@@ -26,6 +26,7 @@
 	<div class="form-group">
 		<@liferay_aui.input
 			name=namespacedFieldName
+			required=required
```

Comment:
Do we really want to make the hidden input have the required validation?

Reply:
Retesting, it looks like having the required here does not affect the field. This can be removed.